ddtrace/commands/ddtrace_run.py

```diff
@@ -147,3 +147,7 @@ def main():
         raise
 
     sys.exit(0)
+
+
+if __name__ == "__main__":
```

Comment:
This makes it easy to use lldb with ddtrace-run

Reply:
Nice